File: python/vidu1_3_1.py
import time

def print_numbers(n):
    for i in range(1, n + 1):
        print(i, end=" ")
    print()

n = 5
start = time.time()  # Bắt đầu đo thời gian

print_numbers(n)  # Chạy thuật toán

end = time.time()  # Kết thúc đo thời gian
elapsed_ms = (end - start) * 1000

print(f"Thời gian (ms): {elapsed_ms:.2f}")

File: C++/vidu1_3_1.cpp
#include <iostream>
#include <chrono>

using namespace std;
using namespace std::chrono;

void printNumbers(int n) {
    for (int i = 1; i <= n; i++) {
        cout << i << " ";
    }
    cout << endl;
}

int main() {
    int n = 5;

    auto start = high_resolution_clock::now(); // Bắt đầu đo thời gian

    printNumbers(n); // Chạy thuật toán

    auto end = high_resolution_clock::now(); // Kết thúc đo thời gian
    auto duration = duration_cast<milliseconds>(end - start).count();

    cout << "Thời gian (ms) = " << duration << endl;

    return 0;
}
